internal/venice-common/src/main/java/com/linkedin/venice/blobtransfer/NettyP2PBlobTransferManager.java

```diff
@@ -54,7 +54,6 @@ public CompletionStage<InputStream> get(String storeName, int version, int parti
         // TODO: add some retry logic or strategy to choose the peers differently in case of failure
         // instanceName comes as a format of <hostName>_<applicationPort>
         String chosenHost = peer.split("_")[0];
-        LOGGER.info("Chosen host: {}", chosenHost);
```

Comment:
Why do we need to remove it? Can we make it debug if that's too much?

Reply:
I removed it on accident when cleaning up my debugging logs, bringing it back